src/api/public/apidocs-new/paths/source_branch.yaml

```diff
@@ -0,0 +1,108 @@
+post:
+  summary: Branch a set of packages based on attributes or on existing request
+  description: Branch a set of packages based on attributes or on existing request.
```

Comment:
What is the point of having a description if it's the same as the summary?

Reply:
I've updated the summaries and descriptions. :slightly_smiling_face: